### DISCUSSION TITLE
[Beta Projects] Ability to reorder (drag and drop) your saved views

### DISCUSSION THREAD

**Discussion by jess-williamson-treespleasegames** [community/community#6486] [Projects and Issues] [answered]
As a user I want to be able to easily reorganise my saved views from left to right. Drag and drop functionality would be easiest.

**Comment by stafyniaksacha** [comment]
Duplicate of [this discussion](https://github.com/github/feedback/discussions/4803).

**Comment by azenMatt** [comment, marked as answer]
Hi @jess-williamson-treespleasegames @stafyniaksacha  - we just released re-orderable view tabs as part of our [recent updates](https://github.blog/changelog/2021-11-18-the-new-github-issues-11-18-update/). 

You can now drag and drop the views into any order you want 😄
